v1/lib/core/__tests__/utils.test.js

```diff
@@ -96,6 +97,34 @@ describe('utils', () => {
     fs.writeFileSync(tempFilePath, 'Lorem ipsum :)');
     expect(utils.getGitLastUpdated(tempFilePath)).toBeNull();
     fs.unlinkSync(tempFilePath);
+
+    // test renaming and moving file
+    const tempFilePath2 = path.join(__dirname, '__fixtures__', '.temp2');
+    fs.writeFileSync(tempFilePath2, 'Lorem ipsum :)');
+
+    shell.exec(`git add ${tempFilePath2}`);
```

Comment:
Don't execute shell commands in tests. Use [Jest mocking](https://jestjs.io/docs/en/mock-functions#using-a-mock-function) to mock out the entire `shell` module and the methods that your method calls.

Reply:
Okay!!! Will try it :D